args/gen.go

```diff
@@ -1,3 +1,5 @@
+// +build -darwin
+
```

Comment:
Why is this needed?

Reply:
When I tested earlier I was getting an empty `curlopts.go` on MacOS.

However, testing again just now it's magically generating correctly again... No idea what happened there.

Removed. Pushed again.